apps/examples/webserver_test/webserver_test_main.c

```diff
@@ -0,0 +1,1308 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <tinyara/config.h>
+#include <tinyara/compiler.h>
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <netdb.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+
+#include <tinyara/version.h>
+#include <netutils/netlib.h>
+
+#include "../../../external/webserver/http_string_util.h"
+#include "../../../external/webserver/http_client.h"
+#include <protocols/webserver/http_err.h>
+#include <protocols/webclient.h>
+
+
+#define MBED_DEBUG_LEVEL 1
+#define WS_TEST_BUF_SIZE 4600
+
+#define WGET_OK                    0
+#define WGET_ERR                   -1
+#define WGET_MSG_CONSTRUCT_ERR     -2
+#define WGET_SOCKET_CONNECT_ERR    -3
+
+#define HTTPSTATUS_NONE            0
+#define HTTPSTATUS_OK              1
+#define HTTPSTATUS_MOVED           2
+#define HTTPSTATUS_ERROR           3
+
+#define ISO_nl                     0x0a
+#define ISO_cr                     0x0d
+#define ISO_space                  0x20
+
+#define WS_TEST_MAXHTTPLINE 200
+#define WS_TEST_MAXMIMESIZE 32
+#define WS_TEST_MAXHOSTNAME 128
+#define WS_TEST_MAXFILENAME 128
+
+#define PRNT(...) \
```

Comment:
can we enable / disable from menuconfig - verbose, debug
include FUNC_EN PRNT and FUNC_EX PRNT also